main.js

```diff
@@ -0,0 +1,177 @@
+const galleryModule = (() => {
```

Comment:
zrobiles jedna niefajna rzecz - dependencje sa brane bezposrednio z obiektu global (np. linie - 122, 134) - czy moglbys to zmienic?

Reply:
Czy dobrze zrozumiałem, że chodzi o zamknięcie całości w osobnym scope'ie ?

Reply:
@lsiedlaczek poprawiłem, otrzymane wartości z dwóch modułów przekazuję teraz w argumencie do  ostatniego - imageProcessorModule. Sprawdź proszę, czy o to chodziło :)